crates/solver/src/main.rs

```diff
@@ -280,6 +281,19 @@ struct Arguments {
     /// but at the same time we don't restrict solutions sizes too much
     #[clap(long, env, default_value = "15000000")]
     simulation_gas_limit: u128,
+
+    /// In order to protect against malicious solvers, the driver will check that settlements prices do not
+    /// exceed a max price deviation compared to the external prices of the driver, if this optional value is set.
+    /// The max deviation value should be provided as a float percentage value. E.g. for a max price deviation
+    /// of 3%, one should set it to 0.03f64
+    #[clap(long, env)]
+    max_settlement_price_deviation: Option<f64>,
+
+    /// This variable allows to restrict the set of tokens for which a price deviation check of settlement
+    /// prices and external prices is executed. If the value is not set, then all tokens included
+    /// in the settlement are checked for price deviation.
+    #[clap(long, env)]
+    token_list_restriction_for_price_checks: Option<Vec<H160>>,
```

Comment:
Nit: why not price_check_tokens?

Reply:
WIth your naming, I the None option would be harder to understand. If one passes in none, for `price_check_tokens` I would guess no tokens are being used.